mobile/library/common/jni/android_jni_utility.cc

```diff
@@ -17,15 +17,15 @@
 bool is_cleartext_permitted(absl::string_view hostname) {
 #if defined(__ANDROID_API__)
   envoy_data host = Envoy::Data::Utility::copyToBridgeData(hostname);
-  JNIEnv* env = Envoy::JNI::get_env();
-  jstring java_host = Envoy::JNI::native_data_to_string(env, host);
+  Envoy::JNI::JniHelper jni_helper(Envoy::JNI::get_env());
+  jstring java_host = Envoy::JNI::native_data_to_string(jni_helper, host);
   jclass jcls_AndroidNetworkLibrary =
       Envoy::JNI::find_class("io.envoyproxy.envoymobile.utilities.AndroidNetworkLibrary");
-  jmethodID jmid_isCleartextTrafficPermitted = env->GetStaticMethodID(
+  jmethodID jmid_isCleartextTrafficPermitted = jni_helper.getEnv()->GetStaticMethodID(
```

Comment:
Your plan is to (incrementally) replace code like this with `jni_helper.GetStaticMethodID(...)`, right?

Reply:
Yup. That's the plan. I will try to make small changes at a time, so it's easier to debug in case something breaks. Once everything has been fully migrated, I'll start removing `JniHelper::getEnv()`.